Doc/c-api/exceptions.rst

```diff
@@ -753,43 +753,45 @@ the variables:
 +-----------------------------------------+---------------------------------+----------+
 | C Name                                  | Python Name                     | Notes    |
 +=========================================+=================================+==========+
-| :c:data:`PyExc_BaseException`           | :exc:`BaseException`            | \(1)     |
-+-----------------------------------------+---------------------------------+----------+
-| :c:data:`PyExc_Exception`               | :exc:`Exception`                | \(1)     |
-+-----------------------------------------+---------------------------------+----------+
 | :c:data:`PyExc_ArithmeticError`         | :exc:`ArithmeticError`          | \(1)     |
```

Comment:
Since that are definitions of `PyExc_* names`, wouldn't be better to use the `c:var` directive?

```
.. c:var:: PyExc_ArithmeticError
```

Perhaps the `index` directive below will be not needed after that.